src/backend/distributed/deparser/deparse_database_stmts.c

```diff
@@ -205,3 +224,88 @@ DeparseAlterDatabaseSetStmt(Node *node)
 
 	return str.data;
 }
+
+
+static void
+AppendCreateDatabaseStmt(StringInfo buf, CreatedbStmt *stmt)
+{
+	appendStringInfo(buf,
+					 "CREATE DATABASE %s",
+					 quote_identifier(stmt->dbname));
+
+	DefElem *option = NULL;
+
+	foreach_ptr(option, stmt->options)
+	{
+		/*If option is template, lc_type, locale or lc_collate, propagation will not be supportted */
+		/* since template database is not stored in the catalog */
```

Comment:
```suggestion
		/*
         * If option is template, lc_type, locale or lc_collate, propagation will
         * not be supportted since those options are not stored in the catalog.
         */
```

Reply:
Fixed